build.gradle

```diff
@@ -13,7 +13,8 @@ buildscript {
       'okio': '2.4.3',
       'okhttp': '4.2.2',
       'moshi': '1.6.0',
-      'protobuf': '0.8.10',
+      'protobuf': '3.11.4',
+      'protobufGradlePlugin': '0.8.10',
       'protoc': '3.0.0',
```

Comment:
a) Let's keep the `protobuf` and `protoc` versions consistent.
e.g.
```
  String protobufVersion = '3.11.0'
  ext.versions = [
      'protobuf': protobufVersion,
      'protobufGradlePlugin': '0.8.10',
      'protoc': protobufVersion,
  ]
```
If we explicitly put the version, such as `3.11.0`, for each key, it'd be nice to have a comment
e.g. `'protoc': '3.11.0', // 'protoc' version should be the same as 'protobuf' version.`

b) Given that grpc-java still uses `3.11.0`, we use the same version as of now? (maybe they have their _own_ reason why they fix the version to 3.11.0 though.)

ref [grpc-java has the same version setting](https://github.com/grpc/grpc-java/blob/v1.27.1/build.gradle#L51-L52)

Reply:
the gradle plugin itself seems to be out of sync with the protobuf versioning.
I will create a hierarchy for the dependencies though